BP/common/religion/religions/00_hellenism.txt

```diff
@@ -323,6 +324,7 @@
 				CreatorHerHim = CHARACTER_HERHIM_IT
 			}
 		}
+		#WtWSMS differences: no raiding, **can't** be reformed, has HOF
```

Comment:
Reformable head of faith, but not preexisting?

Reply:
From WtWSMS Latin Solar:
* There is (and can exist) no religious head.
* Can be reformed
* Can raid infidel neighbors for loot

The Sol Invictus Cult, otoh, should be organized (like Hellenism itself) as it was an official imperial cult for a while. So there's no need to reform it (organized), and I should add a specific head of faith (temporal) instead of letting CK3 auto-generate it.

Reply:
I disagree, by 476 there are no longer any institutions and it is no longer reorganised. Reformation is the process that allows it to be reinstituted. Thus it should be disorganised, reformable and without head. In WtWSMS Latin Solar by default had a temporal and secular head, a bit like the Norse.

Reply:
Alright, will make it unreformed.